src/collective/embeddedpage/browser/embeddedpage.py

```diff
@@ -87,8 +95,16 @@ def process_page(self):
             for link in el.findall('.//head//link'):
                 body.insert(0, link)
             el = body
-        return etree.tostring(el, method='html')
+        data['content'] = etree.tostring(el, method='html')
+        return data
 
     def __call__(self):
-        self.embeddedpage = self.process_page()
+        data = self.process_page()
```

Comment:
@csenger wouldn't it be more straight forward to just add all the code from process_page to __call__? Just wondering.

Reply:
@tisto Rodrigo refactored that out of __call__() cause for the serializer he added. .process_page() is used in the serializer.

Please merge this PR for an release.